examples/fluids/qfunctions/newtonian.h

```diff
@@ -783,4 +784,28 @@ CEED_QFUNCTION(PressureOutflow_Jacobian_Prim)(void *ctx, CeedInt Q,
   return PressureOutflow_Jacobian(ctx, Q, in, out, StateFromY, StateFromY_fwd);
 }
 
+// *****************************************************************************
+// This QFunction implements the Harten Lax VanLeer (HLL) approximate Riemann
+// solver. Taking in two states.
+// *****************************************************************************
+CEED_QFUNCTION_HELPER int Harten_Lax_VanLeer_Flux(void *ctx, CeedInt Q,
```

Comment:
It's not a qfunction, and this will be called separate at each quadrature point (should be inlined by the compiler). The job here is to return a flux, so we can make it a return argument.
```suggestion
CEED_QFUNCTION_HELPER StateConservative Harten_Lax_VanLeer_Flux(NewtonianIdealGasContext gas,
```